wnprc_billing/resources/queries/ehr_billing/invoiceExternalLastRun.sql

```diff
@@ -0,0 +1,28 @@
+SELECT
+  invExternal.rowId,
+  invExternal.invoiceNumber,
+  invExternal.invoiceRunId,
+  invExternal.billingPeriodStart,
+  invExternal.billingPeriodEnd,
+  invExternal.BillingRunDate,
+  invExternal.accountNumber,
+  invExternal.type,
+  invExternal.po_number,
+  invExternal.contact_name,
+  invExternal.address,
+  invExternal.invoiceAmount
+FROM ehr_billing.invoiceExternal invExternal WHERE invExternal.invoiceRunId =(SELECT max(invoiceRunId) from ehr_billing.invoiceExternal)
+
+GROUP BY
```

Comment:
Why is this a grouped query? If it's important to de-dupe, let's do a SELECT DISTINCT instead.

Reply:
Yup, you're right, it's not needed. I've added Distinct. 
Thanks!